backend/python/app/services/implementations/story_service.py

```diff
@@ -570,6 +570,24 @@ def soft_delete_story_translation(self, id):
             self.logger.error(error)
             raise error
 
+    def remove_user_from_story_translation(self, story_translation_id, user_id):
+        try:
+            story_translation = StoryTranslation.query.get(story_translation_id)
+            if not story_translation:
+                raise Exception("Error. Story translation does not exist.")
+            if user_id == story_translation.translator_id:
+                return self.soft_delete_story_translation(story_translation_id)
```

Comment:
nit: strikes me as odd that we're returning `soft_delete_story_translation` here since `soft_delete_story_translation` has a void return and this function also seems to have a void return